content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -266,6 +297,14 @@ visit [MCR Deployment Guide](https://docs.mirantis.com/mcr/20.10/install.html).
 Check the systemd unit named `cri-docker.socket` to find out the path to the CRI
 socket.
 
```

Comment:
here we can just copy the same `#### Overriding the sandbox (pause) image` from the section from `### Docker Engine`